src/app/pages/swap/components/swap-selected-asset-placeholder.tsx

```diff
@@ -0,0 +1,43 @@
+import { FiChevronDown } from 'react-icons/fi';
+
+import { Stack, Text } from '@stacks/ui';
+
+import { SelectedAssetField } from '@app/components/forms/selected-asset-field';
+import { Flag } from '@app/components/layout/flag';
+import { SpaceBetween } from '@app/components/layout/space-between';
+
+import { SwapAmountField } from './swap-amount-field';
+import { SwapSelectedAssetLayout } from './swap-selected-asset.layout';
+
+interface SwapSelectedAssetPlaceholderProps {
+  onChooseAsset(): void;
+  title: string;
+}
+export function SwapSelectedAssetPlaceholder({
+  onChooseAsset,
+  title,
+}: SwapSelectedAssetPlaceholderProps) {
+  return (
+    <SwapSelectedAssetLayout title={title} value="0">
+      <SelectedAssetField height="76px" mb="tight" name="swapAssetFrom">
+        <Flag align="middle" img={<></>} spacing="tight">
+          <SpaceBetween>
+            <Stack
```

Comment:
Why do we need both of these here? Are they doing the same thing?

Reply:
I don't understand why you think they are doing the same thing? The `Stack` is only wrapped around two elements that are inline horizontally on the left side of the `SpaceBetween` ...the amount field is on the right side on it's own.